robot/power.py

```diff
@@ -7,6 +8,18 @@
 LOGGER = logging.getLogger(__name__)
 
 
+# Keep this in sync with `robotd`
+class PowerOutput(enum.Enum):
+    """An enumeration of the outputs on the power board."""
+
+    HIGH_POWER_0 = 0
```

Comment:
We should probably match the names in the docs -- either `H1` or `HIGH_CURRENT_1`.

Reply:
We should match the labelling on the boards surely...?

Reply:
I'm not sure what the boards say, though I think they use `H1`. Do you have a power board you could check with @trickeydan?